clients/src/main/java/org/apache/kafka/clients/admin/RemoveMembersFromConsumerGroupOptions.java

```diff
@@ -34,51 +32,15 @@
 @InterfaceStability.Evolving
 public class RemoveMembersFromConsumerGroupOptions extends AbstractOptions<RemoveMembersFromConsumerGroupOptions> {
 
-    /**
-     * A struct containing member's info.
-     */
-    public static class RemovingMemberInfo {
-        public String memberId;
-        public String groupInstanceId;
-
-        public RemovingMemberInfo(String memberId, String groupInstanceId) {
-            this.memberId = memberId;
-            this.groupInstanceId = groupInstanceId;
-        }
-
-        @Override
-        public boolean equals(Object o) {
-            if (o instanceof RemovingMemberInfo) {
-                RemovingMemberInfo otherMember = (RemovingMemberInfo) o;
-                return this.memberId.equals(otherMember.memberId) &&
-                           this.groupInstanceId.equals(otherMember.groupInstanceId);
-            } else {
-                return false;
-            }
-        }
-
-        @Override
-        public int hashCode() {
-            return Objects.hash(groupInstanceId, memberId);
-        }
-    }
-
-    private Set<RemovingMemberInfo> members;
+    private Set<GroupMember> members;
 
     public RemoveMembersFromConsumerGroupOptions(Collection<String> groupInstanceIds) {
```

Comment:
Do we want a constructor which accepts `Collection<GroupMember>`?

Reply:
I guess for now a list of string should be suffice

Reply:
Oh actually we should use that collection for long term consistency